src/lib/telldus-accessory.js

```diff
@@ -47,6 +48,12 @@ class TelldusAccessory {
     this.log = string => log(`[${this.name}]: ${string}`)
     this.Characteristic = homebridge.hap.Characteristic
     this.Service        = homebridge.hap.Service
+
+    // Support age limit for sensors to avoid stale information.
+    this.isStale = sensor => {
```

Comment:
Correct, this can no longer be undefined since that was handled in the config PR. I'll make the change.